### PR TITLE
Add IBM Copyright Statements

### PR DESCRIPTION
**Pull request by mobasherul**

### CODE REVIEW

**Review by chrisdennis — CHANGES_REQUESTED** [review]
This seems to have gained a bunch of partial Apache headers. It might be legitimate that this becomes Apache licensed... but if we're going to do that then it should be a conscious effort. Probably best to just preserve the existing mess for now?

File: src/main/java/com/tc/object/LiteralValues.java [hunk 1]
```diff
@@ -1,527 +1,527 @@
...
-    final LiteralValues type = valueFor(value);
-    return type.calculateDsoHashCodeForLiteral(value);
-  }
-
-}
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object;
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/object/LiteralValues.java]: you should avoid including reformatting in this PR

File: src/main/java/com/tc/bytes/TCByteBuffer.java [hunk 2]
```diff
@@ -1,165 +1,165 @@
...
-
-  public void checkedOut();
-
-  public BufferPool getBufferPool();
-
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.bytes;
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/bytes/TCByteBuffer.java]: you should avoid including reformatting in this PR

File: src/main/java/com/tc/object/ObjectID.java [hunk 3]
```diff
@@ -1,85 +1,85 @@
...
-  public String toString() {
-    if (toLong() == -1 || getGroupID() == 0) { return super.toString(); }
-    return getIdentifierType() + "=" + "[" + getGroupID() + ":" + getMaskedObjectID() + "]";
-  }
-
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object;
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/object/ObjectID.java]: you should avoid including reformatting in this PR

File: src/main/java/com/tc/object/TraversedReferences.java [hunk 4]
```diff
@@ -1,39 +1,40 @@
...
-   * Walk through the references traversed
-   * @return Iterator&lt;TraversedReference&gt;
-   */
-  public Iterator<TraversedReference> iterator();
-}
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object;
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/object/TraversedReferences.java]: you should avoid including reformatting in this PR

File: src/main/java/com/tc/object/dna/api/DNA.java [hunk 5]
```diff
@@ -1,90 +1,90 @@
...
-}
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object.dna.api;
+
+import com.tc.object.ObjectID;
+
+/**
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/object/dna/api/DNA.java]: you should avoid including reformatting in this PR

File: src/main/java/com/tc/object/dna/api/DNACursor.java [hunk 6]
```diff
@@ -1,67 +1,68 @@
...
-   * except for it's "comparator" field which is treated physically
-   * @return LogicalAction, PhysicalAction, or LiteralAction
-   */
-  public Object getAction();
-
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object.dna.api;
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/object/dna/api/DNACursor.java]: you should avoid including reformatting in this PR

File: src/main/java/com/tc/object/dna/api/DNAEncoding.java [hunk 7]
```diff
@@ -1,74 +1,74 @@
...
-   * @param length The length of the array to encode
-   */
-  public abstract void encodeArray(Object value, TCDataOutput output, int length);
-
-}
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object.dna.api;
```
Comment (aurbroszniowski) [inline src/main/java/com/tc/object/dna/api/DNAEncoding.java]: you should avoid including reformatting in this PR

**Review by aurbroszniowski — CHANGES_REQUESTED** [review]
there's a number of classes that you reformatted, no reformatting of the code should be included in this PR, the git history changed will get confusing

File: src/main/java/com/tc/object/LiteralValues.java (hunk 1 above)
Comment (mobasherul) [reply src/main/java/com/tc/object/LiteralValues.java]: There were lot of files with CRLF line ending which got changed  to LF while doing commit thats' why it's happening

**Pull request merged**